content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,277 @@
+
+### Multiple Python Instances
+
+Tests targeting O3DE can create multiple separate Python interpreter instances during execution, which each load different scripts. Most commonly, there are two instances: the _external interpreter_ and the _Editor interpreter_. The remainder of this section will help determine which environment your test should run in, and how to execute it there.
+
+#### Tests in the external interpreter
+
+The external interpreter runs outside of any active O3DE application. This is the same interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd`, and is best used for tests which involve the following:
+
+* Generic tests that launch applications and send external signals, often with [LyTestTools](/docs/user-guide/testing/lytesttools/).
+* Monitoring application crashes.
+
+To target PyTest in the external interpreter, by default your tests filenames should start with `test_`.
+
+#### Tests in the Editor interpreter
+
+The O3DE Editor internally manages a Python Interpreter, and exposes Editor-specific functionality through a Python bindings library. Although this environment isn't equivalent to launching `o3de/python/python.*`, it uses the same version of the Python interpreter. The Editor interpreter is best used for tests which involve the following:
+
+* Target specific functionality within the Editor, using [EditorPythonBindings and EditorTest](/docs/user-guide/testing/parallel-pattern/).
+* Rely on external crash-handling (editor crash causes a test script crash).
+
+To integrate with the Editor interpreter, create a test which uses `EditorPythonBindings`. These tests **must not** be in a file starting with `test_` or `tests_`, to avoid accidentally registering as failing tests.
+
+{{< note >}}
+PyTest is not used within the Editor interpreter, and as a consequence PyTest functionality is unavailable to tests which run in the Editor interpreter. Avoid dependencies on PyTest fixtures when designing these tests.
+
+EditorTest still uses PyTest to manage tests, and additionally handles external crash monitoring, batching, and parallelism. If this tool does not meet your needs, please reach out with a [feature request](https://github.com/o3de/o3de.org/issues/new/choose) or start a discussion in [Discord](https://discord.gg/p3padwr58u) channel SIG-Testing!
+{{< /note >}}
+
+### Registering a new Python test
+
+Registering a Python-based test is simpler than registering a C++ test. However, it still requires that you define the C++ library that you want to test, which is likely already completed before designing integration tests. The steps below assume the production library is already defined. You can read more about defining production code in [Getting Started with the CMake Build System](/docs/user-guide/build/).
```

Comment:
```suggestion
Registering a Python-based test is simpler than registering a C++ test. However, it still requires that you define the C++ library that you want to test, which you've likely already completed before designing integration tests. The steps below assume the production library is already defined. You can read more about defining production code in the [Build](/docs/user-guide/build/) section.
```

Similar to previous suggestion, is there a more specific section you're referring to? If so, update the link. You can also specify subsections within a page by adding `#` and the name of the subsection, lower-cased and space-separated with hyphens. For example: `/docs/user-guide/build/configure-and-build#configuring-projects`

For now, my suggestion updates the name of the reference link to more accurately describe what users will find when they click on it.

Reply:
I do not want to deep-link into this other section of documentation, and users of this system need to know the content it introduces.